openapi-validator/src/main/java/org/ballerinalang/openapi/validator/ErrorMessages.java

```diff
@@ -22,100 +22,159 @@
  */
 class ErrorMessages {
 
-    static String invalidFilePath(String path) {
```

Comment:
Seems like there are some unused methods in this class, If we don't need them we can remove.

Reply:
We hope to use that error messages in the future implementations